tests/issues/issue-1923.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+return [
+    'issue-1923-A' => [
+        'ua' => 'Mozilla/5.0 (Linux; U; Android 6.0; ru-ru; 5080X Build/MRA58K) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Mobile Safari/537.36',
+        'properties' => [
+            'Comment' => 'Yandex Browser 16.2',
```

Comment:
Comment does not match to Browser. Should be `Android Browser 4.0`.

Reply:
That's why I wanted to run the tests :P Any way fixed both mismatches